modules/hub/infrastructure/helm-values-production.yaml

```diff
@@ -28,6 +28,18 @@ env:
   SERVICE_USER_KEY: 'nnWZrKw7WuE0K1JWb2PHup42xEVrLDnkMTcEbqecF3NeuLcleGl7crJwo2Hc0Uh'
   CHANNEL_MANAGER_ADDRESS: '0xbf2aeab03b82c182599930217272ef0fc5cf662b'
   TOKEN_CONTRACT_ADDRESS: '0x6b01c3170ae1efebee1a3159172cb3f7a5ecf9e5'
+  SHOULD_COLLATERALIZE_URL: 'https://beta.spankchain.com/api/model-signup/should-collateralize/'
```

Comment:
do we want to commit these?

Reply:
Can definitely be removed

Reply:
(and should also be removed on our end)

Reply:
@wolever, is this supposed to be ignored?